rfcs/0044-node-status-backend.md

```diff
@@ -0,0 +1,36 @@
+## Summary
+
+[summary]: #summary
+
+This RFC proposes that we use AWS stack to be the backend of the node status/error system. To be specific by AWS stack, I mean we would use S3 as the storage backend; OpenSearch as the search engine; Kibana as the visualization and charting tool; we could also setup the miniservice that listening on the minaprotocol/node-status on AWS ES2.
+
+## Motivation
+
+[motivation]:#motivation
+
+We need a backend for node staus/error systems. Candidates for the backend include AWS stack and Grafana Loki and LogDNA.
+
+## Implementation
+
+[implementation]:#implementation
```

Comment:
What does the microservice look like? Is it in OCaml? What data does it consume and in what format, what services does it talk to?

Reply:
How does S3, OpenSearch, Kibana, and the microservice all communicate with each other?

Reply:
The mini-service would push to Kenesis Firehose data stream and AWS use this to communicate between OpenSearch, Kibana and S3.